tokio-console/src/config.rs

```diff
@@ -428,6 +481,13 @@ impl Config {
             log_directory: other.log_directory.or(self.log_directory),
             target_addr: other.target_addr.or(self.target_addr),
             log_filter: other.log_filter.or(self.log_filter),
+            warnings: {
+                let mut warns: Vec<KnownWarnings> = other.warnings;
+                warns.extend(self.warnings);
+                warns.sort_unstable();
+                warns.dedup();
+                warns
```

Comment:
Potentially, we might want to store the list of known warnings as a `BTreeSet` rather than as a `Vec`? That way, the properties that the list is sorted and contains no duplicates will be ensured at all times, rather than just when we ensure they are true.

On the other hand, I'm not sure whether that changes how it's deserialized from TOML...so, just a thought.

Reply:
> On the other hand, I'm not sure whether that changes how it's deserialized from TOML...so, just a thought.

`default_values_t` seems to only support the vec as the default value.